client/me/purchases/list/site/index.jsx

```diff
@@ -14,6 +14,7 @@ import SectionHeader from 'components/section-header';
 
 const PurchasesSite = ( { isPlaceholder, name, purchases, slug } ) => {
 	let items, label = name;
+	const siteUrl = slug && slug.replace( /::/g, '/' );
```

Comment:
Got it. Then I suggest instead of `currentValue.domain` we use `siteObject.domain`, because that gets rid of the protocol for you (so you don't have to in render) but should still have the sub-directories.